pillar/heroku/discussions.sls

```diff
@@ -60,6 +61,7 @@
       'MITXPRO_BASE_URL': 'xpro-rc.odl.mit.edu',
       'OCW_ITERATOR_CHUNK_SIZE': 300,
       'OCW_NEXT_AWS_STORAGE_BUCKET_NAME': 'ol-ocw-studio-app-qa',
+      'OCW_NEXT_BASE_URL': 'https://ocwnext-rc.odl.mit.edu/',
```

Comment:
This should actually be live-qa.ocw.mit.edu

Reply:
fixed